.github/workflows/publish-client.yml

```diff
@@ -0,0 +1,165 @@
+---
+# Automatically build and publish the client to the container registry.
+# This workflow is triggered on changes to the client directory.
+
+name: Build and Publish hmi-client
+
+# yamllint disable-line rule:truthy
+on:
+  push:
+    paths: [ 'packages/client/**' ]
+    branches: [ 'main' ]
+    tags: [ '*' ]
+
+jobs:
+  image-name:
+    name: Create image name
```

Comment:
you could use this for the tag generation, it gives you a few more perks but may be overkill here
[Metadata action](https://github.com/docker/metadata-action)

Reply:
After looking at it, yes I agree it is overkill.
Would be nice to have setup for Pantera though